src/compressor/Compressor.h

```diff
@@ -73,9 +73,7 @@ class Compressor {
   QatAccel qat_accel;
 #endif
 
-  union {
-    int zlib_winsize = -15;
-  };
+  int compressor_message = 0;
```

Comment:
IMO better move to ZLibCompressor and access via compress/decompress functions.

Reply:
change type to int32_t to be inline with the subsequent usage from bluestore and rgw?